tests/unit_tests/test_simulation.py

```diff
@@ -21,4 +22,7 @@ def test_check_switch_geo_trigger(simulation):
 
 
 def test_read_k_eds_delta(simulation):
+    old_db_path = simulation.db_path
+    simulation.db_path = str(Path(simulation.db_path).parents[1] / 'tap_reference_db.h5')
     assert simulation.read_k_eds_delta(7) is False
+    simulation.db_path = old_db_path
```

Comment:
How come the `db_path` needs to be updated here?

Reply:
For consistency if we ever add more tests to `test_simulation.py`.